frontend/simple/views/containers/Proposals.vue

```diff
@@ -1,42 +1,48 @@
 <template>
   <div>
-    <!--  Start Original Voting Banner to delete soon -->
-    <voting-banner v-for="proposal in Object.values(proposals)"
-      :proposal="proposal"
-    />
-    <!--  End Original Voting Banner to delete soon -->
-
-    <!-- REVIEW: Not sure about these banners. -->
-
-    <!--
-    <h4 class="title is-size-5 notification is-warning gi-is-banner gi-notify"
-      v-if="groupProposals.notVoted.length"
-    >
-      <i18n>These are waiting for your vote!</i18n>
-    </h4>
-    -->
 
     <i18n class="notification gi-is-banner gi-notify" v-if="allVoted">Cool, you already voted on all proposals.</i18n>
 
     <voting
-      v-for="proposal in groupProposals.notVoted"
+      v-for="proposal in proposalList.notVoted"
       :type="proposal.type"
-      :proposal="proposal.data"
+      :votes="proposal.votes"
+      :value="proposal.value"
+      :member="proposal.member"
+      :originalValue="proposal.originalValue"
+      :ownVote="proposal.ownVote"
+      :isOwnProposal="proposal.isOwnProposal"
+      :initiator="proposal.initiator"
+      :hash="proposal.hash"
```

Comment:
Much better having all props split ✨ !

Do you know if there is a way similar to react, to reuse prop values? We repeat from `:type` to `:hash` 3x exactly the same thing. On React is something like

```js
function commonProps({ types, votes, value }) { // and other props...
  return { types, votes, value }
}

<Voting { ...commonProps(proposal) } :onCloseProposal="handleCloseProposal" />
```

Reply:
The proposal object itself could be passed in, and that seems like a good idea here.

Reply:
@taoeffect on why we expanded the proposal prop see the discussion here: https://github.com/okTurtles/group-income-simple/pull/415#discussion_r193523388 

"I would however unwrap it and have separate props instead of one proposal object, so that props validation can be more helpful and can work as a kind of self-documenting code, making what the component's expecting more explicit"

so I would go towards more self-explanatory, less compact in this case. but it's your call @taoeffect

Reply:
@sandrina-p that is one of the nice features of jsx, to be able to use js syntax sugars like that inside html-looking stuff, that we lost when deciding on vue :( and as far as I see, most workarounds end up passing just one prop, where we lose everything we won with spreading out the proposal object :)

Reply:
@hubudibu the props should already be validated by the messages in the contract chain, no? So I don't think we need to re-validate them here?

Just looking at how the code is turning out, it seems like it would be simpler to pass in the proposal object.

Reply:
I found something! https://github.com/vuejs/vue/issues/4962#issuecomment-280779102 

"You can already do this with `v-bind="currentUser"`."

if this works, our lives will be much easier here :) I'll give this a try!

Reply:
As @hubudibu said, I also agree with splitting a proposal object in different props, so the component itself is more self explanatory. Also, that's the correct way of using "props". Otherwise every component could just receive a "data" prop with everything inside.